### PR TITLE
chore: add to contributor documentation on how to deal with breaking changes

### PR DESCRIPTION
**Pull request by janechu**
# Description

Adds documentation for contributors on how to document breaking changes.

## Motivation & context

This clarifies how contributors can document breaking changes so that a clear migration strategy is outlined.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://microsoft.github.io/fast-dna/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -40,6 +40,21 @@ Before submitting a pull request, [rebase](https://www.atlassian.com/git/tutoria
 
 Learn more about [submitting pull requests](https://microsoft.github.io/fast-dna/docs/en/contributing/working).
 
+### Documenting breaking changes
+
+Make sure to document the migration strategy in a `MIGRATION.md` file in the package(s) that has breaking changes.
```
Comment (chrisdholt) [inline CONTRIBUTING.md]: Assuming we want this in the _root_. Do we want to specify location? Is it helpful to be explicit?

Comment (janechu) [reply CONTRIBUTING.md]: A breaking change only affects the package that the breaking change is in, so it should be localized to packages.

Comment (janechu) [reply CONTRIBUTING.md]: Or did you mean I should provide an example of where it would go?

Comment (chrisdholt) [reply CONTRIBUTING.md]: Yeah, more of that. Is it helpful to show a path like:
```
packages/fast-components-styles-msft/MIGRATION.md
```

Comment (chrisdholt) [reply CONTRIBUTING.md]: not explcitly saying it *is* - more of a question

Comment (janechu) [reply CONTRIBUTING.md]: Added

**Pull request merged**